syfoalerts.yaml

```diff
@@ -108,3 +108,10 @@ spec:
       action: "Sjekk logger, og om det kan ha blitt innført feil"
       sla: respond within 1h, during office hours
       severity: danger
+    - alert: syfosm-error-logging
+      expr: sum(rate(logd_messages_total{log_app=~"syfosmoppgave|syfosmarena|syfosmvarsel|syfosminfotrygd|syfosmregister|syfosmsak|syfosmmottak|syfosmregler",log_level="Error",job="kubernetes-pods"}[5m])) > 0
```

Comment:
Med denne regelen får vi alerts i praksis hver gang en av appene logger en error. I utgangspunktet er jeg enig i at når noe logges som error, så betyr det at vi skal inn og se på det. Samtidig kommer det an på hvor aktiv man er på å logge ting som error, hadde vi hatt en sånn alert for f.eks. syfoservice hadde vi blitt nedlesset i alarmer 😅 Så.. Vil det være mange tilfeller der syfosm-appene logger error, men at det ikke er nødvendig for oss å se på det? 

Jeg er egentlig veldig åpen for å prøve denne litt prinsippfaste grensen, vi kan jo bare justere det hvis det blir for mye.

Reply:
Tror at det ikkje kommer til å skje så ofte.
Tror dei gange det faktisk logges error at vi må se på det.
Evt så kan vi jo bare justere som du seier dersom det blir for mye.

Sjekket i dei ulike appene, hvor mange log.error innsalg vi hadde:
syfosminfotrygd: 1 Error
syfosmmottak: 1 Error
syfosmoppgave: 1 Error
syfosmregler: 1 Error
syfosmarena: 0 Error
syfosmregiser: 5 Error
syfosmvarsel: 1 Error

Reply:
Enig! :)